libs/watt/src/lib/components/picker/datepicker/watt-datepicker.component.ts

```diff
@@ -205,6 +207,25 @@ export class WattDatepickerComponent extends WattPickerBase {
         this.datePlaceholder = this.getPlaceholderByLocale(locale);
         this.rangePlaceholder = this.getRangePlaceholder();
       });
+
+    effect(() => {
+      this.rangeMonthOnlyMode();
+      this.ngControl?.control?.updateValueAndValidity();
+    });
+  }
+
+  override ngAfterViewInit() {
```

Comment:
question: Isn't it dangerous to override this method? The one in the base class does something :)

Reply:
Yes, what was I thinking!? So that was why it was complaining about a missing override keyword 😄 It is so rare to extend classes these days, so totally missed that. Good catch!

973d58b28330f38c02de6df2729caac4ae2b166e